client-js/app/elements/labrad-registry.html

```diff
@@ -2,6 +2,7 @@
 <link rel="import" href="../bower_components/paper-toast/paper-toast.html">
 <link rel="import" href="app-link.html">
 <link rel="import" href="dir-item.html">
+<link href='https://fonts.googleapis.com/css?family=Roboto+Mono' rel='stylesheet' type='text/css'>
```

Comment:
I can move the import to elements.html and it still works, this way the font can be called in the local css without including the import. I guess that wouldn't make the custom elements standalone, but at this point they are pretty interdependent anyhow.